ui/cypress/e2e/buckets.test.ts

```diff
@@ -22,7 +22,7 @@ describe('Buckets', () => {
       const newBucket = '🅱️ucket'
       cy.getByTestID('table-row').should('have.length', 1)
 
-      cy.contains('Create').click()
+      cy.contains('Create Bucket').click()
```

Comment:
no `getByTestID` ?

Reply:
this test was just modified bc there were now two create buttons on the page, but i can change it to use the get by test id